src/main/java/manageezpz/logic/commands/EditTaskCommand.java

```diff
@@ -118,20 +122,73 @@ public CommandResult execute(Model model) throws CommandException {
         }
     }
 
+    /**
+     * If the value after a prefix is empty, return false.
+     * Else, return true.
+     */
+
+    boolean ensureFormatCompliance(HashMap<String, Boolean> prefixStatusHash,
```

Comment:
For good programming practice, do declare whether this method is public or private